Solutions/Corvus.Extensions.Newtonsoft.Json/Corvus/Extensions/Json/Internal/JsonNetPropertyBag.cs

```diff
@@ -41,15 +41,20 @@ public JsonNetPropertyBag(IDictionary<string, object> dictionary, JsonSerializer
                 throw new System.ArgumentNullException(nameof(dictionary));
             }
 
```

Comment:
Can't you compactify this to 

```
this.serializerSettings = serializerSettings ?? throw new Blah();
```

Reply:
I could, and I generally do if left to my own devices, but in cases where the code is already doing the other style, I generally follow suit. And since this particular constructor was already using the `if` style, I did that for the 2nd argument too. Although now I look at it I realise it's a bit odd that the two constructors in this class are doing it differently.

I think that unless I were to do a sweep to change it across the whole project, I'm not going to submit a further change in this PR just to make those two ctors match.